### DISCUSSION TITLE
Administration panel  suggestion

### DISCUSSION THREAD

**Discussion by GeoSot** [spatie/laravel-event-sourcing#300] [Q&A]
Hello Guys

This package, is is really extraordinary, as the application using the event sourcing pattern 

As I am trying to put it in one new project and while planning, I fell of in front of an admin panel decision I have to do.
Most of the 'admin panel packages'  makes the crud processes out of the box, but in straight communication with the proper Eloquent model / Db

So what is your approach or your suggestion over this? Have you used this package in addition of any ready admin packages?
How can we make them cooperate ?

**Comment by brendt** [comment]
The golden rule with event sourcing is that you only event source what's really needed. Admin CRUD operations probably shouldn't be event sourced. Here's some more reading on the topic: https://stitcher.io/blog/combining-event-sourcing-and-stateful-systems

**Reply by gkakosouris** [reply]
That's a great read, but I think that the question still stands: how did you approach Order administration? Would it even be possible to use something like Nova to manage event-sourced contexts?

**Reply by brendt** [reply]
You could potentially hook into Nova or model events and use those as the trigger for stored events, or store those events themselves. 

I would never do that myself though. We're working on a large event sourced project with a very complex admin backend, and made a clear border between the non event sourced admin backend, and the event sourced parts (shopping cart, invoicing, stuff like that).
